.github/workflows/translate.yml

```diff
@@ -0,0 +1,61 @@
+name: "Update translations"
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 3 * * *"
+    
+permissions: {}
+
+defaults:
+  run:
+    shell: pwsh
+
+jobs:
+  update-translations:
+    permissions:
+      # for git push
+      contents: write
+    runs-on: ubuntu-latest
+
+    env:
+      TX_TOKEN: ${{ secrets.TX_TOKEN }}
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: recursive
+          ref: ${{ github.ref }}
+
```

Comment:
For the desktop client, we have a remove old step.
https://github.com/owncloud/client/blob/master/.github/workflows/translate.yml#L33
This is needed as we have a requirement in place that we require at least 75% to be translated. Configured in https://github.com/owncloud/client/blob/master/translations/.tx/config .

The threshold only prevented pulls of new languages, but that means that translations below the threshold remained in the repo (we had a 7 years old in the desktop repo).
Do we have a threshold configured for this project?

Reply:
added config key